projects/client/RabbitMQ.Client/src/client/impl/Command.cs

```diff
@@ -75,7 +75,7 @@ public Command(MethodBase method, ContentHeaderBase header, byte[] body)
         }
 
         public byte[] Body { get; private set; }
-
+        
```

Comment:
Please be aware of and remove whitespace changes like these.

Reply:
Yeap, sorry, missed it 😞